app/src/main/java/com/pwhs/quickmem/presentation/app/notification/NotificationViewModel.kt

```
@@ -0,0 +1,159 @@
+                        sendErrorEvent(result.message ?: "Failed to mark notification as read")
+                    }
+                }
+            }
+        }
+    }
+
+    private fun isValidUUID(uuid: String): Boolean {
+        return try {
+            UUID.fromString(uuid)
+            true
+        } catch (e: IllegalArgumentException) {
```

detekt

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)